app/views/issues/show_reviewers.html.haml

```diff
@@ -10,3 +10,17 @@
         Recenzent: 
         = link_to review.reviewer, review.person
         %br
+  %p{style: "margin-top: 20px;"}
+    Recenzenci z Uniwersytetu Jagiellońskiego: #{@count_uj} (
```

Comment:
Ten kod powinien być w modelu, a odpowiednie dane powinny być przekazane przez kontroler. To jest niezgodne z wzorcem MVC - logika powinna być tylko w modelu - nigdy w widoku (szablonie).